.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,3 +5,6 @@
 - [ ] Passes `isort . && black . && mypy . && flake8`
 - [ ] User visible changes (including notable bug fixes) are documented in `whats-new.rst`
 - [ ] New functions/methods are listed in `api.rst`
+
+
+<sub>By default, the upstream dev CI is disabled on pull request and push events. You can override this behavior by adding a `[test-upstream]` tag in the git commit’s title.</sub>
```

Comment:
not sure if "commit title" is a well-known term. Also, it would probably help to state that this has to be done for every commit the CI is supposed to run on.

Maybe something like this instead?
```suggestion
<sub>By default, the upstream dev CI is disabled on pull request and push events. You can override this behavior per commit by adding a `[test-upstream]` tag in the first line of the commit message.</sub>
```